test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3109,6 +3109,58 @@ def setup_rnn_tests():
 
 setup_rnn_tests()
 
+# Unary floating ufunc tests
+# Tests that integral arguments are properly cast to float
+unary_floating_ufuncs = (
+    'cos',
+    'asin',
+    'ceil',
+    'expm1',
+    'floor',
+    'log',
+    'log10',
+    'log1p',
+    'log2',
+    'sin',
+    'sinh',
+    'sqrt',
+    'trunc',
+    'atan',
+    'cosh',
+    'exp',
+    'tan',
+    'tanh',
+    'erf',
+    'erfc',
+    'erfinv',
+    'lgamma',
+    'digamma',
+    'rsqrt',
+    'sigmoid',
+    'acos'
```

Comment:
Is this list complete?

Reply:
Yes. I cross-checked the list. This list looks complete to me (except `mvlgamma` - which is under discussion).